.github/workflows/benchmark-flow.yml

```diff
@@ -0,0 +1,117 @@
+name: Run a Benchmark Flow
+
+on:
+  workflow_call:
```

Comment:
I assume this is manual triggering ?

Reply:
This contains the actual logic for benchmarks within CI. Meaning we can trigger benchmarks in different manners (PR, weekly, merges to master/version) but the actual logic/steps is contained within this file, and all others invoke it.